src/resource.ts

```diff
@@ -180,11 +186,18 @@ export default class Resource<T = any> {
     // will coalesc them into one.
     if (!this.inFlightRefresh) {
 
+      const headers: { [name: string]: string } = {
+        Accept: this.contentType ? this.contentType : this.client.getAcceptHeader()
+      };
+
+      if (this.preferPushRels.size > 0) {
+        headers['Prefer-Push'] = Array.from(this.preferPushRels).join(' ');
```

Comment:
Is it likely that servers will implement both? I guess it doesn't hurt to send both, and just kinda see what comes back?

Reply:
Yea exactly. The library supports either transparently, so it's up to the server to decide which they want (and hopefully not do both!)

I'm not sure if this is the right choice yet, but if this will cause issues for people I'm sure they'll speak up.